android/src/main/kotlin/com/dns_technologies/mlkit_scanner/models/InitialScannerParameters.kt

```diff
@@ -0,0 +1,18 @@
+package com.dns_technologies.mlkit_scanner.models
+
+/** Describes initial parameters to initialize scanner  */
+data class InitialScannerParameters(
+    val initialZoom: Double? = null,
```

Comment:
`Initial` по отношению к данным класса является внешним контекстом. То есть он никак не используется внутри класса. Стоит назвать:
```suggestion
    val zoom: Double? = null,
    val cropArea: RecognizeVisorCropRect? = null,
```